### PR TITLE
Pods 5584

### PR DESCRIPTION
**Pull request by brownbonnie**

### CODE REVIEW

File: app/identifiers/establishers/partnership/contact/EnterEmailId.scala [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+case class EnterEmailId(index: Int) extends TypedIdentifier[String] {
+  override def path: JsPath =
+    EstablishersId(index).path \ EnterEmailId.toString
+}
+
+object EnterEmailId {
+  override lazy val toString: String =
+    "partnershipEmail"
+
+  def collectionPath(index: Int): JsPath =
```
Comment (madhuranakate) [inline app/identifiers/establishers/partnership/contact/EnterEmailId.scala:32]: collectionPath not required

File: app/identifiers/establishers/partnership/contact/EnterPhoneId.scala [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+case class EnterPhoneId(index: Int) extends TypedIdentifier[String] {
+  override def path: JsPath =
+    EstablishersId(index).path \ EnterPhoneId.toString
+}
+
+object EnterPhoneId {
+  override lazy val toString: String =
+    "partnershipPhone"
+
+  def collectionPath(index: Int): JsPath =
```
Comment (madhuranakate) [inline app/identifiers/establishers/partnership/contact/EnterPhoneId.scala:32]: collectionPath not required

File: app/helpers/cya/establishers/partnership/EstablisherPartnershipContactDetailsCYAHelper.scala [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        id = EnterPhoneId(index),
+        message = Message("messages__enterPhone_cya_label", establisherName).resolve,
+        url = Some(routes.EnterPhoneController.onPageLoad(index, CheckMode).url),
+        visuallyHiddenText = Some(msg"messages__enterPhone__cya__visuallyHidden".withArgs(establisherName))
+      ))).flatten
+
+    rowsWithDynamicIndices(rowsWithoutDynamicIndices)
+
+  }
+}
```
Comment (madhuranakate) [inline app/helpers/cya/establishers/partnership/EstablisherPartnershipContactDetailsCYAHelper.scala:65]: Tests are missing for CYA helper, navigator and DataCompletion

**Review by madhuranakate — CHANGES_REQUESTED** [review]

File: app/identifiers/establishers/partnership/contact/EnterEmailId.scala (hunk 1 above)
Comment (jigneshkumarpatel) [reply app/identifiers/establishers/partnership/contact/EnterEmailId.scala]: remove collectionPath

**Review by madhuranakate — APPROVED** [review]

File: app/helpers/cya/establishers/partnership/EstablisherPartnershipContactDetailsCYAHelper.scala (hunk 3 above)
Comment (brownbonnie) [reply app/helpers/cya/establishers/partnership/EstablisherPartnershipContactDetailsCYAHelper.scala:65]: Unit tests added

**Pull request merged**